plate_mapper/plate_mapper.py

```diff
@@ -73,16 +75,21 @@ def plate_mapper(input_f, barseq_f, output_f, names_f=None, special_f=None):
                 elif int(v) != cols+1:
                     raise ValueError('Error: column headers are not '
                                      'incremental integers.')
+            # get plate ID (trailing numeric part)
+            for i in reversed(l[0]):
```

Comment:
re comes bundled with the python standard library - so importing it shouldn't be a problem.